src/dataflow/src/server/storage_state.rs

```diff
@@ -103,7 +103,71 @@ impl<'a, A: Allocate, B: StorageCapture> ActiveStorageState<'a, A, B> {
         match cmd {
             StorageCommand::CreateSources(sources) => {
                 // Nothing to do at the moment, but in the future prepare source ingestion.
-                for (id, (description, _since)) in sources.into_iter() {
+                for (id, (description, _since), ts_bindings) in sources.into_iter() {
```

Comment:
Would it be possible to make this be a method, rather than in-line code? Back when it was "add binding" it was .. localized complexity. But it's now next to other higher-level actions that imo it would help to not mask. For example, a per-`GlobalId` method that does the `ts_bindings` work for that source.